Makefile

```diff
@@ -246,5 +246,17 @@ socketcan:
 update_codegen:
 	@python make/git_fetch.py -folder=libraries/codegen-tooling -user=uw-midsun -repo=codegen-tooling-msxiv -tag=latest -file=codegen-tooling-out.zip
 
+.PHONY: test_python
+test_python:
```

Comment:
`test_python` is a little bit long for a thing people will be typing over and over again, maybe `pytest` would be better (in the same vein as `pylint`). If you change it to that also change `test_all_python` -> `pytest_all`.